src/main/java/audio/AudioMaster.java

```diff
@@ -0,0 +1,89 @@
+package audio;
+
+import org.lwjgl.BufferUtils;
+import org.lwjgl.openal.*;
+
+import java.nio.ByteBuffer;
+import java.nio.IntBuffer;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.lwjgl.openal.ALC10.*;
+import static org.lwjgl.openal.EXTEfx.ALC_MAX_AUXILIARY_SENDS;
+
+/**
+ * Main audio center: equivocal to graphics.Window but for initializing audio stuff.
+ * @author HilbertCurve
+ */
+public class AudioMaster {
+
+    private static List<AudioSource> sources = new ArrayList<>();
+    private static AudioMaster instance;
+    private static long device;
+
+    private AudioMaster() {
+
+    }
+
+    public static AudioMaster get() {
+        if (instance == null) {
```

Comment:
I suggest making the inner part of instance==null synchronized - to prevent two get calls possibly running parallel